contribs/gmf/src/directives/map.js

```diff
@@ -21,9 +22,8 @@ goog.require('ngeo.mapDirective');
  */
 gmf.mapDirective = function() {
   return {
-    scope: {},
-    bindToController: {
-      'map': '=gmfMapMap'
+    scope: {
+      'getMapFn': '&gmfMapMap'
     },
```

Comment:
Just curious why you eventually used scope instead of bindToController (by the way don't we need a ``bindToController: true`` line?)?